default.nix

```diff
@@ -32,17 +37,24 @@ in project // {
     packages = ps: with ps; [ amazonka amazonka-test amazonka-gen ];
 
     tools = {
-      cabal = "3.2.0.0";
-      cabal-fmt = "0.1.5.1";
-      shellcheck = "0.7.1";
-      ormolu = "0.3.0.1";
+      cabal-fmt = {
+        inherit (pkgs.cabalProject) index-state;
+        version = "0.1.5.1";
+      };
```

Comment:
Can we use `pkgs.haskellPackages.cabal-fmt` in `buildInputs`, or is that version too old?

Reply:
If we use `pkgs.haskellPackages.cabal-fmt` we need to potentially download another GHC from the cache - by using `tools` it ensures it uses the same (patched) GHC from haskell.nix.